### ISSUE TITLE
evohome-config greyed out and not visible in pallette

### ISSUE THREAD

**Issue by tstaddon** [dehsgr/node-red-contrib-evohome#8]
I'm running the plugins on a vanilla install of a new Pi 4b.

Node-RED version: v1.2.6
Node.js version: v12.20.0
Linux 5.4.79-v7l+ arm LE

When I install the evohome plugin I can access Control and Status but Config is not visible anywhere.

**Comment by dehsgr** [comment]
Hi @tstaddon,

I would like you to provide some screen shots, to clarify. You should be able to create your configuration note from within control and status node(s):
![image](https://user-images.githubusercontent.com/4125820/103562569-988a8d80-4ebb-11eb-93e7-b37ecc7e9040.png)
![image](https://user-images.githubusercontent.com/4125820/103562584-9d4f4180-4ebb-11eb-9055-1509d05e86fe.png)

**Comment by tstaddon** [comment]
Thanks. I could already see exactly what you have on the screenshot but as the documentation said there were three nodes included I was expecting to see three distinct nodes in the palette. If I hit the pencil I can edit the control node per shown here. 

![20210104184833](https://user-images.githubusercontent.com/51123466/103568566-ad682080-4ebd-11eb-8f98-182fa50428f4.png)

In my defence, I am getting some truly weird stuff at the moment. I have only the one Pi for some IoT work and haven't yet done anything except flash the SD card, boot it up, run the updates and then test a completely fresh debug node using the timestamp to create the payload (so, just the two "Hello, I'm totally new to this" buttons). 

Once that worked I added the Evohome Status node and that's when it all started to go weird. At the moment I can't see any evidence that the Status node is doing anything at all - and every now and again Node-Red decides to insist "node has undeployed changes" despite me having already used all four options in the deploy menu at least three times since the last time I edited anything.

Perhaps I'm missing a dependency somewhere.
